If /etc/mtab is a symbolic link autofs will fail to mount
via ssh and emit the error fuse: unknown option(s): `-n'
This error is fixed by allowing '-n' to pass through.
--- lib/mount.c
+++ lib/mount.c
@@ -97,6 +97,7 @@
 	FUSE_OPT_KEY("rootcontext=",		KEY_KERN_OPT),
 	FUSE_OPT_KEY("max_read=",		KEY_KERN_OPT),
 	FUSE_OPT_KEY("user=",			KEY_MTAB_OPT),
+	FUSE_OPT_KEY("-n",			KEY_MTAB_OPT),
 	FUSE_OPT_KEY("-r",			KEY_RO),
 	FUSE_OPT_KEY("ro",			KEY_KERN_FLAG),
 	FUSE_OPT_KEY("rw",			KEY_KERN_FLAG),
